nevergrad/optimization/lamcts/turbo_1/functions.py

```diff
@@ -0,0 +1,281 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+# verify
```

Comment:
what is this "verify" about?

Reply:
Because not sure that we want this license.